packages/preview/graph-gen/0.1.0/README.md

```diff
@@ -0,0 +1,63 @@
+# graph-gen
+
+Graph view for your typst notes
+
+![alt text](imgs/graph-example.png)
```

Comment:
Can you write an actual alternative text please? Something like "Part of large graph. Vertices are represented by various shapes, in different colors, and are labelled with the part of the document they correspond to like "Finals", "RSA", "Signature schemes", "Key generation". The edges are also colored."